v3/internal/assetserver/bundledassets/runtime.debug.js

```diff
@@ -859,8 +854,8 @@ var Window = class _Window {
    * @param {boolean} resizable - Whether the window should be resizable
    * @return {Promise<void>}
    */
-  SetResizable(resizable) {
-    return this[caller](SetResizableMethod, { resizable: resizable });
+  SetResizable(resizable2) {
```

Comment:
resizable2?

Reply:
New esbuild version attempting to avoid some identifier collisions I think.
I saw that in other places too after the latest esbuild update.